src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/TargetsTest.java

```diff
@@ -321,8 +321,8 @@ public void testTargetsSUSE() {
         WorkloadInventoryReportModel report = reports.get(0);
         Assert.assertNotNull(report.getRecommendedTargetsIMS());
         Assert.assertEquals(2, report.getRecommendedTargetsIMS().size());
-        Assert.assertTrue(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains("RHV".toLowerCase())));
-        Assert.assertTrue(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains("OSP".toLowerCase())));
+        Assert.assertTrue(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains(WorkloadInventoryReportModel.TARGET_RHV.toLowerCase())));
+        Assert.assertTrue(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains(WorkloadInventoryReportModel.TARGET_OSP.toLowerCase())));
         Assert.assertFalse(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains("RHEL".toLowerCase())));
```

Comment:
```suggestion
        Assert.assertFalse(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains(WorkloadInventoryReportModel.TARGET_RHEL.toLowerCase())));
```
We have changed the text of the RHEL target we're searching for.